spec/ruby_basics_spec.rb

```diff
@@ -0,0 +1,68 @@
+require 'spec_helper'
+require_relative '../lib/ruby_basics.rb'
+
+
+describe "ruby" do
+
+  describe "#division" do
+    it 'given two numbers returns quotient of the first number divided by the second number' do
+      num1 = 42
+      num2 = 7
+
+      expect(division(num1,num2)).to eq(6)
+    end
+  end
+
+  describe '#assign_variable' do
+    it 'takes an argument of a persons name and assign it to a variable name' do
+      value = "Bob"
+
+      expect(assign_variable(value)).to eq(value)
+    end
+  end
+
+  describe '#argue' do
+    it 'accepts an argument and return that argument as is' do
+      phrase = "I'm right and you are wrong!"
+
+      expect(argue(phrase)).to eq(phrase)
+    end
+  end
+
+  describe '#greeting' do
+    it 'takes two arguments' do
+      greeting = "Hi there, "
+      name = "Bobby!"
+
+      expect{ greeting(greeting, name) }.to_not raise_error
```

Comment:
Eh, that's not really testing what you're aiming at here. I think the test can be better written:

Options:

* Assign to a global, give them that variable in the starter code?
* Something else.